tests/unit/src/test/scala/tests/ReferenceLspSuite.scala

```diff
@@ -38,11 +38,11 @@ class ReferenceLspSuite extends BaseRangesSuite("reference") {
           |  val z: Int = y.a
           |  val param: String = a.A.param(arg = 2)
           |}
-          |""".stripMargin,
-        preInitialized = { () => server.didOpen("a/src/main/scala/a/A.scala") }
```

Comment:
So this would hang, because it's run before sending `initialized` message.

Reply:
Ah gotcha, that makes sense.